packages/story-editor/src/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -139,9 +154,14 @@ function PageAttachmentPanel() {
   const handleChangeUrl = useCallback(
     (value) => {
       _setUrl(value);
-      debouncedUpdate({ url: value });
+      setIsInvalidUrl(false);
+      const urlWithProtocol = withProtocol(value.trim());
+      const valid = isValidUrl(urlWithProtocol);
```

Comment:
Shouldn't the invalid URL flag be set if not valid here?

```suggestion
      const urlWithProtocol = withProtocol(value.trim());
      const valid = isValidUrl(urlWithProtocol);
      setIsInvalidUrl(!valid);
```

Reply:
`isValidUrl` doesn't seem to be working reliably so it doesn't work fully as expected when doing that. Tried that initially but it didn't have consistent behaviour like this.